Library/OcSmbiosLib/SmbiosPatch.c

```diff
@@ -143,7 +147,19 @@ PatchBiosInformation (
   SMBIOS_OVERRIDE_V (Table, Standard.Type0->EmbeddedControllerFirmwareMajorRelease, Original, NULL, NULL);
   SMBIOS_OVERRIDE_V (Table, Standard.Type0->EmbeddedControllerFirmwareMinorRelease, Original, NULL, NULL);
 
+  AppleDetected = FALSE;
+
+  CHAR8 *Vendor = SmbiosGetString(Table->CurrentPtr, (Table->CurrentPtr).Standard.Type0->Vendor);
```

Comment:
Watch out codestyle please. EDK II prefers C89 style of variable declaration. That is to say to declare all variables at the beginning of functions, and then initialise them where necessary.

Also, there should be a space character between function names and the left parenthesis. e.g. `SomeFunction (Args);`